esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -28,9 +40,9 @@ The |RTW| performs the following steps:
      The ``configure.py`` script from the |Rose| app
   :Details:
      Runs once at the beginning of the workflow, immediately after the
-     successful completion of the ``install_cold`` job, and then at the
-     start of each cycle, after the successful completion of the
-     previous cycle's compare jobs
+     successful completion of the ``clone_latest_esmval`` job, and then at the
+     end of each cycle, after the successful completion of the compare jobs
+     to prepare for the start of the next testing cycle.
```

Comment:
`configure` should run at the start of each cycle after `clone_latest_esmval` has completed; in the case where the next cycle doesn't happen for a week, the clone could potentially be out of date by then.

Reply:
Addressed in 3a4f54b0ee0801777290eb90af6f71ce0a9f31a6.

Reply:
Addressed in https://github.com/ESMValGroup/ESMValTool/pull/2871/commits/58151b5646ee7416cc6f623047976f28c8ede220 instead.